spec/routes_spec.rb

```diff
@@ -14,6 +14,32 @@
 describe 'Application routes' do
   before(:all) { MiqProductFeature.seed_features }
 
+  def error_message
+    <<~ERROR.gsub
```

Comment:
Think you missed the tailing `.gsub`
```suggestion
    <<~ERROR
```

Reply:
Same below